src/app/ReadClient.h

```diff
@@ -351,11 +351,12 @@ class ReadClient : public Messaging::ExchangeDelegate
      *  Re-activate an inactive subscription.
      *
      *  When subscribing to LIT-ICD and liveness timeout reached and OnResubscriptionNeeded returns
-     * CHIP_ERROR_LIT_SUBSCRIBE_INACTIVE_TIMEOUT, the read client will move to the InactiveICDSubscription state and
-     * resubscription can be triggered via OnActiveModeNotification().
+     *  CHIP_ERROR_LIT_SUBSCRIBE_INACTIVE_TIMEOUT, the read client will move to the InactiveICDSubscription state and the
+     *  subscription is put on hold, resubscription can be triggered via OnActiveModeNotification().
+     *  When client encounters various timeouts, whether that be exchange timeout, CASE setup timeout, or lack of getting a report
+     * on time, client will move to FailedICDSubscription state and subscription is still retrying with back-off algorithm, the
```

Comment:
This documentation (which claims this happens for "LIT-ICD") very much does not match the code.

Reply:
this change is removed, and unnecessary.